registry/time/helpers.go

```diff
@@ -0,0 +1,34 @@
+package time
+
+import (
+	"testing"
+	"time"
+)
+
+// ComputeTimeFromFormat returns a time.Time object from the given date.
+func ComputeTimeFromFormat(date any) time.Time {
+	switch date := date.(type) {
+	case time.Time:
+		return date
+	case *time.Time:
+		return *date
+	case int64:
+		return time.Unix(date, 0)
+	case int:
+		return time.Unix(int64(date), 0)
+	case int32:
+		return time.Unix(int64(date), 0)
+	}
+
+	// otherwise, fallback to the current time
+	return time.Now().Local()
+}
+
+// ForceTimeLocal temporarily sets [time.Local] for test purpose.
+func ForceTimeLocal(t *testing.T, local *time.Location) {
+	t.Helper()
+
+	originalLocal := time.Local
+	time.Local = local
+	t.Cleanup(func() { time.Local = originalLocal })
+}
```

Comment:
As a test helper, I prefer to declare it in a test file or in pesticide package to prevent import "testify" on the build tree

Reply:
I don't get what you said, this function doesn't use testify :thinking:

Reply:
The function `ForceTimeLocal` are only used for test so moving it to the `pesticide` package to have clear separation between tests and lib usage. 

> to prevent import "testify"
A brain freeze, I don't wanna know why I have say that, because `t.Helper()` are from `testing` and not `testify`.. Sorry